### DISCUSSION TITLE
does mimir support opensdb data reporting to the platform?

### DISCUSSION THREAD

**Discussion by z7658329** [grafana/mimir#6631] [Help and support]
![image](https://github.com/grafana/mimir/assets/5474723/caf72aa6-c72d-4a9d-b40a-6cf702c2384a)

can users send data to distributor by api, rather then use promethus? how many protocols mimir supports reporting ？

**Comment by pstibrany** [comment]
Mimir supports [Prometheus remote-write protocol](https://prometheus.io/docs/concepts/remote_write_spec/), and [OTLP/HTTP protocol](https://opentelemetry.io/docs/specs/otlp/#otlphttp) (metrics only). OTLP ingestion is still experimental in Mimir.
